compat/server.browser.js

```diff
@@ -1,4 +1,9 @@
-export {
+import { renderToString as preactRenderToString } from 'preact-render-to-string';
+
+export const renderToString = preactRenderToString;
+export const renderToStaticMarkup = preactRenderToString;
+
+export default {
 	renderToString,
-	renderToString as renderToStaticMarkup
-} from 'preact-render-to-string';
+	renderToStaticMarkup
+};
```

Comment:
What @marvinhagemeister is referring to is that you're not just adding a default export, but changing the style of the existing exports. Simply append the default export change, no need to alter existing exports.

Reply:
Understood, i'm doing it